src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7234/explicitly.smaxage.and.authorization/connect/server.rpt

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newProxyConnectRef ${nuklei:newReferenceId()} # external scope
+property date ${http_cache:date()} # shared property
+
+accept await ROUTED_PROXY
+        "nukleus://target/streams/http-cache"
+        option nukleus:route ${newProxyConnectRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+accepted
+
+read nukleus:begin.ext ${http:header(":scheme", "http")}
+                       ${http:header(":method", "GET")}
+                       ${http:header(":path", "/path")}
+                       ${http:header(":authority", "localhost:8080")}
+                       ${http:header("authorization", "SOME_AUTH_HEADER_VALUE")}
+
+connected
+
+read closed
+
+write nukleus:begin.ext ${http:header(":status", "200")}
+                        ${http:header("content-length", "11")}
+                        ${http:header("date", date)}
+                        ${http:header("last-modified", date)}
+                        ${http:header("cache-control", "s-maxage=15")}
+write "hello world"
```

Comment:
LGTM, but the cache will inject (I believe) a generated weak etag, if one is not provided in the response.  I would suggest providing one

Reply:
et all

Reply:
Done.